src/App.tsx

```diff
@@ -21,6 +42,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(e) => selectPerPage(e)}
+            value={perPage}
           >
             <option value="3">3</option>
```

Comment:
Would be great to create an array with possible options and iterate over it, to not repeat the same code for each option

Reply:
ok, got it, thanks